cartridge-cli.lua

```diff
@@ -1210,8 +1301,18 @@ local function build_application(dir)
     end
 
     -- post build
-    if fio.path.exists(fio.pathjoin(dir, POSTBUILD_SCRIPT_NAME)) then
-        run_hook(dir, POSTBUILD_SCRIPT_NAME)
+    if opts.deprecated_build_flow then
```

Comment:
Let's define a global variable for it to avoid ugly propagation of this flag via options

Reply:
I don't like using global variables, it's not a constant.
I think it's better to pass options explicitly.